lib/price_calculator.rb

```diff
@@ -0,0 +1,37 @@
+require "terminal-table"
+
+require_relative './cart'
+require_relative './invoice'
+
+class PriceCalculator 
+
+    attr_reader :items
+
+    def initialize
```

Comment:
initialize method should only initialize values. Think of it as a constructor, with regards to Javascript.